docs/ref/contrib/gis/install/index.txt

```diff
@@ -347,7 +347,7 @@ running macOS.  Because MacPorts still builds the software from source,
 
 Summary::
 
-    $ sudo port install postgresql93-server
+    $ sudo port install postgresql10-server
```

Comment:
Would it make sense to make this the latest available release, i.e. 13? If users are starting out it feels like they should be pointed to the latest stable release where possible.

Reply:
As far as I'm aware `13` is not available on MacPorts.

Reply:
Yeah. Looks like 12 is the latest on MacPorts, so we should probably go with that. Homebrew has 13. But IANAMU* so :man_shrugging: .

\* = I am not a Mac user. :joy:

Reply:
me neither :shrug:

Reply:
I'm not a Mac User but I can see PostgreSQL 13 on MacPorts
https://ports.macports.org/port/postgresql13/

Reply:
Let's leave it. I will confirm with Carlton in the next week.

Reply:
IANAM**P**U 😄 

As per @paulxnet's https://github.com/django/django/pull/13894#discussion_r558183709 looks like it's there or will be. 
(I'd imagine folks can work it out…) 

👍 from me to the general idea of the pointing to latest version. (No doubt it'll be another few years before anyone updates this again so...)